src/databricks/labs/ucx/hive_metastore/grants.py

```diff
@@ -135,6 +135,9 @@ def _set_owner_sql(self, object_type, object_key):
         return f"ALTER {object_type} {escape_sql_identifier(object_key)} OWNER TO `{self.principal}`"
 
     def _apply_grant_sql(self, action_type, object_type, object_key):
+        if "DENIED" in action_type:
+            action_type = action_type.replace("DENIED_", "")
+            return f"DENY `{action_type}` ON {object_type} {escape_sql_identifier(object_key)} TO `{self.principal}`"
```

Comment:
we need to wrap the action type in DENY, because `DENY SELECT` throws an error. ``DENY `SELECT` `` works

Reply:
Why it's not a code comment? 😉 add, otherwise someone would remove backticks